validation-test/compiler_crashers_2/6235b4422a78fa73.swift

```diff
@@ -0,0 +1,6 @@
+// {"kind":"typecheck","signature":"swift::rewriting::RewriteContext::getRelativeTermForType(swift::CanType, llvm::ArrayRef<swift::rewriting::Term>)","signatureAssert":"Assertion failed: (result.back().getKind() != Symbol::Kind::Shape), function getRelativeTermForType"}
+// RUN: not --crash %target-swift-frontend -typecheck %s
+struct a < b > {
+  func
+    c < each d where (repeat each d , b) == b>()
+}
```

Comment:
This looks like https://github.com/swiftlang/swift/issues/84490.

Reply:
It is yeah, I added it locally before filing the bug, let me add a link to it